### ISSUE TITLE
Docker desktop MacOs doesn't install macOS

### ISSUE THREAD

**Issue by vinu2003** [docker/for-mac#4692]
- [X ] I have tried with the latest version of my channel (Stable or Edge)
  - [X ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior
Docker for Mac OS installed and run successfully.

### Actual behavior
Docker Desktop needs privileged access.
Docker Desktop needs privileged access to install its networking components and links to 
the Docker apps.

You will be asked for your password.

Typing in the password doesn't install.

### Information
  - macOS Version: Catalina version 10.15.3
<img width="1383" alt="Screen Shot 2020-06-22 at 12 40 11 am" src="https://user-images.githubusercontent.com/31263558/85227518-633bb800-b421-11ea-81e2-79fa5c569f00.png">

**Comment by vinu2003** [comment]
I used the latest stable version for linux in my Ubuntu server and it works fine. Not sure what could be wrong here.

**Comment by vinu2003** [comment]
Tried installing using "brew install docker"
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ brew install docker
Updating Homebrew...
==> Downloading https://homebrew.bintray.com/bottles/docker-19.03.11.catalina.bottle.tar.gz
Already downloaded: /Users/vinodhinibalusamy/Library/Caches/Homebrew/downloads/37340a2af8c7b8e853f07a0898851f04977be6b7a730e67535b61bcb41bbabf1--docker-19.03.11.catalina.bottle.tar.gz
==> Pouring docker-19.03.11.catalina.bottle.tar.gz
==> Caveats
Bash completion has been installed to:
  /usr/local/etc/bash_completion.d

zsh completions have been installed to:
  /usr/local/share/zsh/site-functions
==> Summary
🍺  /usr/local/Cellar/docker/19.03.11: 14 files, 74.8MB
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ 
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ 
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ 
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ docker version
Client: Docker Engine - Community
 Version:           19.03.11
 API version:       1.40
 Go version:        go1.14.3
 Git commit:        42e35e6
 Built:             Tue Jun  2 14:01:31 2020
 OS/Arch:           darwin/amd64
 Experimental:      false
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
vinodhinis-MacBook-Pro:~ vinodhinibalusamy$ 

Looks like i should be getting all the docker-machine manually installed here ? 

Any help is appreciated why the Docker desktop doesn't get installed properly.

**Comment by vinu2003** [comment]
I have had issues with hypervisor app driver earlier when i was trying out minkube installation which required fixes and reboot.

And I noticed this above docker problem not happening anymore.

But docker desktop installation has got nothing to do with hypervisor. 
Couldn't reproduce this problem anymore locally. Couldn't trace what setup in my macbook caused this above issue.

Closing this issue for now.
